.gitignore

```diff
@@ -1,2 +1,5 @@
 *~
 target/
+.gigaide/
+.idea/
+guacamole-client.iml
```

Comment:
These changes should not be included as part of this change. If we need to add entries to .gitignore, those should be handled via a specific Jira issue and pull request.